src/dparse/stack_buffer.d

```diff
@@ -20,14 +22,21 @@ struct StackBuffer
         static if (is(T == class))
             static assert(T.sizeof == size_t.sizeof);
 
+        static if (hasIndirections!T)
+            while (_length % size_t.sizeof != 0)
+                put(ubyte(1));
```

Comment:
The D garbage collector requires pointers to be aligned to the machine word size, otherwise it will not "see" them.

I'm not sure if it was strictly necessary in this case, but, it seemed prudent to add it.